src/documents/views/StudyFilesListView.js

```diff
@@ -52,9 +52,16 @@ const filterFiles = (fileList, filters) => {
     obj.node.fileType.includes(filters.typeFilterStatus),
   );
   // Filter by tags
-  filteredList = filteredList.filter(obj =>
-    obj.node.tags.join(',').includes(filters.tagFilterStatus),
-  );
+  if (filters.tagFilterStatus.length > 0) {
+    filteredList = filteredList.filter(obj =>
+      filters.tagFilterStatus.includes('untagged')
```

Comment:
I suppose you just need to acknowledge that the user may make a tag of their own called `untagged` and may then get back those tags if they choose the `-- untagged --` filter.

Reply:
Maybe don't allow creating their own tag called "untagged".